SPECS-SIGNED/kernel-signed/kernel-signed.spec

```diff
@@ -10,7 +10,7 @@
 Summary:        Signed Linux Kernel for %{buildarch} systems
 Name:           kernel-signed-%{buildarch}
 Version:        5.15.137.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Please add a corresponding changelog entry in the `%changelog` section for this `-2` release increment